content/docs/dev/develop/rawkv/checksum.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Checksum
+description: Learn how to use RawKV's Checksum API.
+menu:
+    "dev":
+        parent: RawKV-dev
+        weight: 2
```

Comment:
```suggestion
        weight: 5
```

To keep the same order as in introduction. Please do a favor to fix the order of "TTL" and "CAS".